doc/api/tls.md

```diff
@@ -723,7 +723,8 @@ The `callback` function, if specified, will be added as a listener for the
     extension.
   * `checkServerIdentity(servername, cert)` {Function} A callback that
     provides  an override for checking the server's hostname against the
-    certificate. Should return an error if verification fails. Returns `undefined` if passing.
+    certificate. Should return an error if verification fails. Returns
+    `undefined` if passing.
```

Comment:
'Returns `undefined` if passing' still is weird.  Maybe 'The callback should return `undefined` when the server identity is accepted'?  Still not great, but better.